testhelpers/docker.go

```diff
@@ -96,17 +110,19 @@ func PushImage(dockerCli dockercli.CommonAPIClient, ref string, auth string) err
 	return nil
 }
 
+//SeedDockerVolume only works with Linux daemons as Windows only mounts volumes for started containers
 func SeedDockerVolume(t *testing.T, srcPath string) string {
 	volumeName := "test-volume-" + RandString(10)
 	containerName := "test-volume-helper-" + RandString(10)
+	volumeHelperImage := "alpine"
```

Comment:
Since this helper can only be used on Linux, I made the image name local. Eventually, when Windows implements this functionality, this function will also be ported.